python/sglang/srt/layers/moe/topk.py

```diff
@@ -158,6 +158,7 @@ def biased_grouped_topk_impl(
     num_expert_group: int = 0,
     topk_group: int = 0,
     n_share_experts_fusion: int = 0,
+    routed_scaling_factor: float = 2.5,
```

Comment:
nit: wondering whether we should make it default as `nan` or `None`, because `2.5` is very specific to DeepSeek. If in some place of code we accidentally forget to pass in the variable, we will get a 2.5 and hard to debug. On the other hand, if we set to none, then if one day another model enables fusion but we forget to pass in value, we will get a big error, and we immediately know it is a bug from where.